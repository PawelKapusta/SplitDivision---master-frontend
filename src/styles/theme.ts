import { DefaultTheme } from "styled-components";

export const theme: DefaultTheme = {
    palette: {
        white: "#fff",
        black: "#000",
        background1: "#0F1624",
        gold: "#eaaf57",
    },
    breakpoints: {
        sm: "screen and (max-width: 640px)",
        md: "screen and (max-width: 768px)",
        lg: "screen and (max-width: 1024px)",
        xl: "screen and (max-width: 1280px)",
    },
};
