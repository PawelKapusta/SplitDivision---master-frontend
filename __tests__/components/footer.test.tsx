import { render } from "@testing-library/react";
import Footer from "@components/footer/index";
import { mockTheme } from "../mocks/theme";
import { ThemeProvider } from "styled-components";

describe("footer component", () => {
    it("renders correctly", () => {
        const { container } = render(
            <ThemeProvider theme={mockTheme}>
                <Footer />
            </ThemeProvider>,
        );
        expect(container).toMatchSnapshot();
    });
});
