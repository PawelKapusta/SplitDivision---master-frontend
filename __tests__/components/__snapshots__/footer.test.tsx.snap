// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component renders correctly 1`] = `
<div>
  <div
    class="MuiBox-root css-5g3grw"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-1xu1mzw-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 css-1unik2p-MuiGrid-root"
        >
          <div
            class="sc-9d0763-0 iKorOP"
          >
            <p
              class="sc-9d0763-2 ehJfRo"
            >
              components.footer.aboutUs.title
            </p>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.aboutUs.careers
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.aboutUs.investors
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.aboutUs.blog
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 css-1unik2p-MuiGrid-root"
        >
          <div
            class="sc-9d0763-0 iKorOP"
          >
            <p
              class="sc-9d0763-2 ehJfRo"
            >
              components.footer.topCities.title
            </p>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.topCities.cracow
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.topCities.sanDiego
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.topCities.london
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.topCities.chicago
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 css-1unik2p-MuiGrid-root"
        >
          <div
            class="sc-9d0763-0 iKorOP"
          >
            <p
              class="sc-9d0763-2 ehJfRo"
            >
              components.footer.services.title
            </p>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.services.development
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.services.stage
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.services.production
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 css-1unik2p-MuiGrid-root"
        >
          <div
            class="sc-9d0763-0 iKorOP"
          >
            <p
              class="sc-9d0763-2 ehJfRo"
            >
              components.footer.social.title
            </p>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.social.instagram
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.social.twitter
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.social.facebook
            </a>
            <a
              class="sc-9d0763-1 gGMNM"
              href="#"
            >
              components.footer.social.linkedin
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
